src/registrar/assets/js/copy-summary.js

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
@CocoByte I don't think the button is copying properly. When I copied details for this request on backup (he-hundred.gov), and pasted into a document, it looked like this: 

Recommendation:
Organization Type: federal
Requested Domain: 163
Existing website(s): 
Rationale:Alternate Domain(s): 
Submitter: 566
Senior Official: 565
Additional Contact(s): 

- For some fields, it's displaying numbers rather than the actual text.
- "Additional contacts," "existing websites," and "alternate domains" appear blank even though there's info for those fields in that request. 
- "Alternate domains" should also  appear on its own line.

Reply:
should be fixed now!